packages/strapi-admin/ee/admin/containers/Roles/CreatePage/Permissions/README.md

```diff
@@ -0,0 +1,377 @@
+# Technical documentation
+
+Here we will cover how the `<Permissions />` component works.
+This component is in charge of managing the permissions of the admin panel, it has 4 children one child for each tab.
```

Comment:
Is "children" the appropriate term here?

Reply:
What do you think would be a better term?